internal/kokoro/vet.sh

```diff
@@ -31,23 +31,25 @@ goimports -l . 2>&1 | tee /dev/stderr | (! read)
 
 # Runs the linter. Regrettably the linter is very simple and does not provide the ability to exclude rules or files,
 # so we rely on inverse grepping to do this for us.
-golint ./... 2>&1 | ( \
-  grep -v "gen.go" | \
-  grep -v "disco.go" | \
-  grep -v "exported const DefaultDelayThreshold should have comment" | \
-  grep -v "exported const DefaultBundleCountThreshold should have comment" | \
-  grep -v "exported const DefaultBundleByteThreshold should have comment" | \
-  grep -v "exported const DefaultBufferedByteLimit should have comment" | \
-  grep -v "error var Done should have name of the form ErrFoo" | \
-  grep -v "exported method APIKey.RoundTrip should have comment or be unexported" | \
-  grep -v "exported method MarshalStyle.JSONReader should have comment or be unexported" | \
-  grep -v "UnmarshalJSON should have comment or be unexported" | \
-  grep -v "MarshalJSON should have comment or be unexported" | \
-  grep -vE "\.pb\.go:" || true) | tee /dev/stderr | (! read)
-
-staticcheck -go 1.9 ./... 2>&1 | ( \
-  grep -v "SA1019" | \
-  grep -v "S1007" | \
-  grep -v "error var Done should have name of the form ErrFoo" | \
-  grep -v "examples" | \
-  grep -v "gen.go" || true) | tee /dev/stderr | (! read)
+golint ./... 2>&1 | (
+  grep -v "gen.go" |
+    grep -v "disco.go" |
```

Comment:
Indenting looks off here.

Reply:
grrr, my bash formatter wanted this I guess... I will revert.